Makefile

```diff
@@ -23,7 +23,7 @@ SHELL:=/usr/bin/env bash
 #
```

Comment:
(had to add the comment somewhere on a changed filed)

* Please update the Go version in v1.1-to-v1.2.md in l.12
* let's bump dependabot.yml and release.yml to 1.18

Note (talked to Nabarun about it already, but for others): we'll keep golangci-lint on Go 1.17 until the have a release with go 1.18 and the gci fix

Reply:
Done! Updated the remaining places.